src/pages/index.js

```diff
@@ -1,43 +1,26 @@
 import React from "react"
 import styled from "@emotion/styled"
 import tw from "tailwind.macro"
+import { graphql, Link } from "gatsby"
 
 import { Layout } from "../components/Layout"
 import { Seo } from "../components/Seo"
 import Banner from "../components/Banner"
 import Header from "../components/Header"
 import Skew from "../components/Skew"
-import { BackgroundCard, DefaultCard, CardDeck } from "../components/Card"
+import Button from "../components/Button"
+import { EmbededPlayer } from "../components/EmbededPlayer"
+import { Container } from "../components/Container"
 
 import logo from "../images/dev_plebs_heads.png"
-import fridayNightDeploys from "../images/podcast_cover_neon.png"
-
-const IndexPage = () => {
-  const FeatuedDeck = (
-    <>
-      <DeckHeader size="md" align="left" fontWeight="light-bold" color="#000">
-        Episode 1: Horrible Bosses
-      </DeckHeader>
-      <p>Cast: Keith, Phil</p>
-      <iframe
-        title="Friday Night Deploys"
-        id="multi_iframe"
-        src="https://www.podbean.com/media/player/multi?playlist=http%3A%2F%2Fplaylist.podbean.com%2F6414558%2Fplaylist_multi.xml&vjs=1&kdsowie31j4k1jlf913=76aa996d66ba97005063b3c6e68595688eca9990&size=430&skin=&episode_list_bg=%23ffffff&bg_left=%23000000&bg_mid=%230c5056&bg_right=%232a1844&podcast_title_color=%23c4c4c4&episode_title_color=%23ffffff&auto=0&download=1&show_playlist_recent_number=10&pbad=1"
-        frameborder="0"
-        scrolling="no"
-        width="100%"
-        height="432"
-        allowfullscreen=""
-      ></iframe>
-    </>
-  )
+
+const IndexPage = ({ data }) => {
+  const { fridayNightDeployEpisode } = data
```

Comment:
It's a pretty long prop name but that doesn't matter that much 😜

Reply:
GOOD CAUSE I DON'T CARE